apko/private/apko_image.bzl

```diff
@@ -163,7 +164,8 @@ def apko_image(name, contents, config, tag, output = "oci", architecture = None,
         for _ in native.glob([resolved_json_name]):
             apko_run(
                 name = name + ".resolve",
-                args = ["resolve", config_label.package + "/" + config_label.name],
+                args = ["resolve", "$(execpath {})".format(config), "--output={}".format(lock_json_name)],
```

Comment:
I'm surprised it works out of the box and you don't need to call

```https://docs.aspect.build/rulesets/aspect_bazel_lib/docs/expand_make_vars/``` manually. 

Have you seen any piece of documentation that acutally says it's supported ?

Reply:
I think it's because `args` is subject to make vars substitution: https://bazel.build/reference/be/common-definitions#common-attributes-binaries

Reply:
Done.